app/src/main/java/org/mozilla/fenix/components/Utilities.kt

```diff
@@ -39,4 +40,8 @@ class Utilities(
     val customTabIntentProcessor by lazy {
         CustomTabIntentProcessor(sessionManager, sessionUseCases.loadUrl, context.resources)
     }
+
```

Comment:
I'd recommend moving this directly the root Component class: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/components/Components.kt

See also: https://github.com/mozilla-mobile/fenix/commit/4922e5d09df650a1bf952dbd2a6652fdaa4669c3#diff-6a1dd8505f3ec4f786426dccbb4a5ca6 (where adding to utils caused a perf. problem).

I think it's also cleaner as otherwise more and more stuff will be added to "utils" and it will eventually become messy. We could also rename `Utilities` to `IntentProcessors` to prevent that.

Reply:
I moved this out and renamed Utilities to IntentProcessors so it's more clear utils shouldn't be added there :)